config/openshift/v4_10_exp/translate.go

```diff
@@ -295,3 +314,75 @@ func validateMCOSupport(mc result.MachineConfig, ts translate.TranslationSet) re
 	}
 	return cutil.TranslateReportPaths(r, ts)
 }
+
+// translateKdump converts OpenShift.Kdump field to other core fields.
+func (c *Config) translateKdump() (r report.Report) {
+	kdump := &c.OpenShift.Kdump
+	if !kdump.Enabled {
+		return r
+	}
+
+	// Enable kdump.service
+	unit := reflect.New(reflect.TypeOf(c.Systemd.Units).Elem()).Elem()
```

Comment:
Reflection shouldn't be needed in this PR.  Instead, see `processBootDevice()` for a good way to implement this.  The idea is to build a `types.Config` data structure, then use `MergeTranslatedConfigs()` to merge it (as the parent config) with the remainder of the user-provided config (as the child config).